test/Microsoft.ML.Tokenizers.Tests/TitokenTests.cs

```diff
@@ -84,6 +89,31 @@ public async void TestTokenizerCreation()
             }
         }
 
+        [Fact]
+        public async void TestTokenizerUsingExternalVocab()
+        {
+            string tokenizerDataFileName = Utils.CreateTemporaryFile("tiktoken");
+            try
+            {
+                await Utils.DownloadFile(@"https://openaipublic.blob.core.windows.net/encodings/cl100k_base.tiktoken", tokenizerDataFileName);
+            }
+            catch
+            {
+                // Tolerate network issues.
```

Comment:
Instead of this can we do a retry with back-off?  I think there are some helpers in runtime we use for that, not sure if they're available here or not.

Reply:
I couldn't find something I can use but I have implemented one. I am still tolerating the failure even after retries.